### PR TITLE
Fix some issues in README

### PR DESCRIPTION
**Pull request by koenrh**
- Correct a few spelling errors.
- Use correct Travis build status badge.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,15 +1,15 @@
 # EmailCorrect
 
-E-mail correct is an utility library for auto correcting an user's e-mail address.
+Email correct is a utility library for auto-correcting a user's email address.
 
-[![Build Status](https://travis-ci.org/jessedijkstra/writing-api.svg?branch=master)](https://travis-ci.org/jessedijkstra/writing-api)
```
Comment (bramschulting) [inline README.md:5]: 🙈

**Review by bramschulting — APPROVED** [review]

**Review by soudy — APPROVED** [review]
:+1:

**Review by Klaasvaak — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (jessedijkstra) [reply README.md:5]: :'(

Comment (jessedijkstra) [reply README.md:5]: How did the writing-api repository end up here btw?

**Pull request merged**